src/index.html

```diff
@@ -4,13 +4,43 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no,
+       initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="https://github.com/Dimoshua/layout_search-bar-airbnb.git"
+      method="post"
+      class="search-bar"
+    >
+      <div data-qa="big">
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="input input--big"
```

Comment:
you can combine `input` and `input--big` styles - check CHECKLIST P2
```suggestion
          class="input"
```